src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -575,11 +573,17 @@ private DockerTransientNode doProvisionNode(DockerAPI api, DockerClient client,
             node.setDockerAPI(api);
             finallyRemoveTheContainer = false;
             return node;
-        } finally {
```

Comment:
Code style:  The rest of this code uses
```
} catch(...) {
```
whereas you've changed this to
```
}
catch(...) {
```
making this bit inconsistent with the rest of the file.
Same applies to `} finally {` too.

While I try not to let my opinions on "which style is better" get in the way of progress :grin: but I do care about consistency.
...also, if you change the code style of your new code to be consistent with the existing style, it'll make for less changes overall, which is no bad thing.